source/edge-server/connect.txt

```diff
@@ -10,4 +10,147 @@ Connect to an Edge Server
    :depth: 2
    :class: singlecol
 
-This is a placeholder page.
+Once the Edge Server is :ref:`configured <edge-server-configure>` and 
+:ref:`running <start-and-stop-edge-server>`, you can connect to it from 
+a client application.
+
+.. _edge-server-connect-from-client:
+
+Connect to the Edge Server from a Client
+----------------------------------------
+
+To connect to the Edge Server from a client, your app must:
+
+- Use HTTP over port 80.
+- Set the Sync URL to the public accessible DNS address you set in the 
+  Edge Server :ref:`config's <edge-server-install-and-configure>` ``hostname`` 
```

Comment:
possible to link directly to that Complete the Edge Server Configuration Details step?

Reply:
Alas, seems we can't link directly to a step.